<!DOCTYPE html>
<html>

<style>
seq {font-family:Monaco, "Courier New";
  font-size:0.9em;
  text-indent: 2.7em;
}

align.small {
    font-family:Monaco, "Courier New";
    font-size:0.9em;
    line-height:65%;
}

align.long {
  font-family:Monaco, "Courier New";
  font-size:0.7em;
  line-height:70%;
}

s5 {color:#6baed6;}
s7 {color:#fc9272;}
p5 {color:#08519c;}
p7 {color:#a50f15;}
me {color:#969696;}
tso {color:#2ca25f;}
cbc {color:#f768a1;}
umi {color:#807dba;}
t7 {color:blue;}
pe1 {color:#bcbddc;}
pe2 {color:#9ebcda;}
w1 {color:#f03b20;}
h3 {font-family:verdana;}

</style>

<head>
<title>sci-RNA-seq</title>
</head>
<body>

<h1><a href="http://science.sciencemag.org/content/357/6352/661" target="_blank">sci-RNA-seq</a></h1>

<span style="font-family:verdana; font-size:1.1em;"><p>The sci-RNA-seq uses the combinatorial indexing to identify single cells without single cell isolation. Two-level indexing (RT barcode + PCR barcodes (i5 + i7)) or three-level indexing (RT barcode + PCR barcodes (i5 + i7) + Tn5 barcodes) can be used. Three-level indexing is a bit more difficult since you need to assemble many indexed Tn5 transposomes. Here, two-level indexing strategy is demonstrated.</p></span>

<br></br>

<h2>Adapter and primer sequences:</h2>
<seq>
<p>Barcoded RT primer: 5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>TTTTTTTTTTTTTTTTTTTTTTTTTTTTTTVN -3'</p>
<p>Nextera Tn5 binding site (19-bp Mosaic End (ME)): 5'- <me>AGATGTGTATAAGAGACAG</me> -3'</p>
<p>Nextera N/S5xx primer entry point (s5): 5'- <s5>TCGTCGGCAGCGTC</s5> -3'</p>
<p>Nextera N7xx primer entry point (s7): 5'- <s7>GTCTCGTGGGCTCGG</s7> -3'</p>
<p>Illumina P5 Primer: 5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5>[i5]<w1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</w1> -3'</p>
<p>Illumina P7 Primer: 5'- <p7>CAAGCAGAAGACGGCATACGAGAT</p7>[i7]<s7>GTCTCGTGGGCTCGG</s7> -3'</p>
<p>Read 1 sequencing primer: 5'- <w1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</w1> -3'</p>
<p>Index 1 sequencing primer (i7): 5'- <me>CTGTCTCTTATACACATCT</me><s7>CCGAGCCCACGAGAC</s7> -3'</p>
<p>Read 2 seuquencing primer: 5'- <s7>GTCTCGTGGGCTCGG</s7><me>AGATGTGTATAAGAGACAG</me> -3'</p>
</seq>


<br></br>

<h2>Step-by-step library generation</h2>
<h3>(1) Anneal Barcoded RT primer to mRNA in fixed cells and reverse transcription using MMLV <i>in situ</i>:</h3>
<pre>
<seq>
5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(T)<sub>30</sub>VN---------->
                                                  (A)<sub>n</sub> BXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX -5'
</seq>
</pre>

<h3>(2) Pool all wells, and re-distribute into wells in a new plate, and perform RNaseH and DNA Pol I based second strand synthesis:</h3>
<pre>
<seq>
5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(dT)VXXXXXXXXXXXXXXXXXXXXXXX -3'
3'- <w1>TGCTGCGAGAAGGCTAGA</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(pA)BXXXXXXXXXXXXXXXXXXXXXXX -5'
</seq>
</pre>

<h3>(3) Add 5ng genomic DNA as carrier, and use Illumina standard Nextera tagmentation on double stranded cDNA plus genomic DNA (will create 9-bp gap):</h3>
<img src="../data/tn5_dimer.svg" alt="Tn5 dimer" style="width:800px;height:450px;">
<pre>
<seq>
<i>Product 1 (s5 at both ends, not amplifiable due to the use of Illumina P5/P7 Primer, see the next step):</i>

5'- <s5>TCGTCGGCAGCGTC</s5><me>AGATGTGTATAAGAGACAG</me>XXXXXXXXXXXX...XXX         <me>CTGTCTCTTATACACATCT</me>
                  <me>TCTACACATATTCTCTGTC</me>         XXX...XXXXXXXXXXXX<me>GACAGAGAATATGTGTAGA</me><s5>CTGCGACGGCTGCT</s5> -5'



<i>Product 2 (s7 at both ends, not amplifiable due to the use of Illumina P5/P7 Primer, see the next step):</i>

5'- <s7>GTCTCGTGGGCTCGG</s7><me>AGATGTGTATAAGAGACAG</me>XXXXXXXXXXXX...XXX         <me>CTGTCTCTTATACACATCT</me>
                   <me>TCTACACATATTCTCTGTC</me>         XXX...XXXXXXXXXXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7> -5'



<i>Product 3 (different s5 and s7 at both ends, not amplifiable, due to the use of Illumina P5/P7 Primer, see the next step):</i>

5'- <s5>TCGTCGGCAGCGTC</s5><me>AGATGTGTATAAGAGACAG</me>XXXXXXXXXXXX...XXX         <me>CTGTCTCTTATACACATCT</me>
                  <me>TCTACACATATTCTCTGTC</me>         XXX...XXXXXXXXXXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7> -5'



<i>Product 4 (s5 at one end, 3' of cDNA at the other end, not amplifiable, due to the use of Illumina P5/P7 Primer, see the next step):</i>

5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(dT)VXXXXXXXXXXXX...XXX         <me>CTGTCTCTTATACACATCT</me> -3'
3'- <w1>TGCTGCGAGAAGGCTAGA</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(pA)BXXXXXXXXXXXX...XXXXXXXXXXXX<me>GACAGAGAATATGTGTAGA</me><s5>CTGCGACGGCTGCT</s5> -5'



<i>Product 5 (s7 at one end, 3' of cDNA at the other end, the only amplifiable product, see the next step):</i>

5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(dT)VXXX...XXX         <me>CTGTCTCTTATACACATCT</me> -3'
3'- <w1>TGCTGCGAGAAGGCTAGA</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(pA)BXXX...XXXXXXXXXXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7> -5'

</seq>
</pre>

<h3>(4) 72 degree gap fill-in (the first cycle in Nextera PCR):</h3>
<pre>
<seq>
5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(dT)VXXX...XXXXXXXXXXXX<me>CTGTCTCTTATACACATCT</me><s7>CCGAGCCCACGAGAC</s7> -3'
3'- <w1>TGCTGCGAGAAGGCTAGA</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(pA)BXXX...XXXXXXXXXXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7> -5'
</seq>
</pre>

<h3>(5) Adding Illumina P5/P7 Primers for library amplification:</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5>[i5]<w1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</w1>------>
                                                5'- <w1>ACGACGCTCTTCCGATCT</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(dT)VXXX...XXXX<me>CTGTCTCTTATACACATCT</me><s7>CCGAGCCCACGAGAC</s7> -3'
                                                3'- <w1>TGCTGCGAGAAGGCTAGA</w1><umi>[8-bp UMI]</umi><cbc>[10-bp RT barcode]</cbc>(pA)BXXX...XXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7> -5'
                                                                                                                          <---------<s7>GGCTCGGGTGCTCTG</s7>[i7]<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(6) Final library structure:</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5>NNNNNNNNNN<w1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</w1><umi>NNNNNNNN</umi><cbc>NNNNNNNNNN</cbc>(dT)VXXX...XXXX<me>CTGTCTCTTATACACATCT</me><s7>CCGAGCCCACGAGAC</s7>NNNNNNNNNN<p7>ATCTCGTATGCCGTCTTCTGCTTG</p7> -3'
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5>NNNNNNNNNN<w1>TGTGAGAAAGGGATGTGCTGCGAGAAGGCTAGA</w1><umi>NNNNNNNN</umi><cbc>NNNNNNNNNN</cbc>(pA)BXXX...XXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7>NNNNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
             <p5>Illumina P5</p5>              i5     <w1>This bit is Truseq adapter</w1>     <umi>8bp UMI</umi>   <cbc>10bp RT</cbc>        cDNA             <me>ME</me>              <s7>s7</s7>           i7        <p7>Illumina P7</p7>
                                                                                      <cbc>barcode</cbc>
</align>
</pre>


<h2>Library sequencing:</h2>

<h3>(1) Add read 1 sequencing primer to sequence the first read (bottom strand as template, these are the UMI and RT barcodes, 18 cycles):</h3>
<pre>
<align class="long">
                                       5'- <w1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</w1>----------------->
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5>NNNNNNNNNN<w1>TGTGAGAAAGGGATGTGCTGCGAGAAGGCTAGA</w1><umi>NNNNNNNN</umi><cbc>NNNNNNNNNN</cbc>(pA)BXXX...XXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7>NNNNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(2) Add Index 1 sequencing primer to sequence i7 index (bottom strand as template, 10 cycles):</h3>
<pre>
<align class="long">
                                                                                                         5'- <me>CTGTCTCTTATACACATCT</me><s7>CCGAGCCCACGAGAC</s7>--------->
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5>NNNNNNNNNN<w1>TGTGAGAAAGGGATGTGCTGCGAGAAGGCTAGA</w1><umi>NNNNNNNN</umi><cbc>NNNNNNNNNN</cbc>(pA)BXXX...XXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7>NNNNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(3) Folds over and sequence the second index (i5 index) (bottom strand as template, 10 cycles):</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5>--------->
3'- <p5>TTACTATGCCGCTGGTGGCTCTAGATGTG</p5>NNNNNNNNNN<w1>TGTGAGAAAGGGATGTGCTGCGAGAAGGCTAGA</w1><umi>NNNNNNNN</umi><cbc>NNNNNNNNNN</cbc>(pA)BXXX...XXXX<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7>NNNNNNNNNN<p7>TAGAGCATACGGCAGAAGACGAAC</p7> -5'
</align>
</pre>

<h3>(4) Cluster regeneration, add Read 2 sequencing primer to sequence the second read (top strand as template, this is the cDNA read, 52 cycles):</h3>
<pre>
<align class="long">
5'- <p5>AATGATACGGCGACCACCGAGATCTACAC</p5>NNNNNNNNNN<w1>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</w1><umi>NNNNNNNN</umi><cbc>NNNNNNNNNN</cbc>(dT)VXXX...XXXX<me>CTGTCTCTTATACACATCT</me><s7>CCGAGCCCACGAGAC</s7>NNNNNNNNNN<p7>ATCTCGTATGCCGTCTTCTGCTTG</p7> -3'
                                                                                                       <-----<me>GACAGAGAATATGTGTAGA</me><s7>GGCTCGGGTGCTCTG</s7> -5'
</align>
</pre>

<br></br>

</body>
</html>
